.github/workflows/conventional-commits.yml

```diff
@@ -1,16 +1,17 @@
 ---
 name: Conventional Commits
-on:  # yamllint disable-line rule:truthy
+on:
   pull_request:
-    types: [opened, reopened, edited, synchronize]
-  workflow_dispatch:  # yamllint disable-line rule:empty-values
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
 jobs:
   lint-pull-request:
```

Comment:
I think we should set the permissions to limit access for the action:

permissions:
  pull-requests: read